src/Storage/Controllers/InstancesController.cs

```diff
@@ -476,7 +478,7 @@ public async Task<ActionResult<Instance>> Delete(int instanceOwnerPartyId, Guid
             [FromRoute] int instanceOwnerPartyId,
             [FromRoute] Guid instanceGuid)
         {
-            (Instance instance, _) = await _instanceRepository.GetOne(instanceOwnerPartyId, instanceGuid);
+            (Instance instance, _) = await _instanceRepository.GetOne(instanceGuid, true);
```

Comment:
Maybe add a reason for why includeDataElements are true:
`// Fetching data elements is needed because the action returns the entire instance`

Reply:
The must interesting thing to address related to this is how to maintain the two instance properties lastchanged and readstatus. In many use cases the data elements are fetched only to make sure that lastchanged and readstatus are set the same way as with the Cosmos repo. The lastchanged semantics is not obvious - the value in the json doc in the db is not the value used externally. One solution could be to always update the two properties directly on the instance for every data element write operation. Then the semantics of the includeDataelements parameter on GetOne would be obvious.